trace-forward/src/Trace/Forward/Acceptor.hs

```diff
@@ -0,0 +1,35 @@
+{-# LANGUAGE LambdaCase #-}
+
+-- | This top-level module will be used by the acceptor application.
+--   Acceptor application asks 'TraceObject's from the forwarder application.
+module Trace.Forward.Acceptor
+  ( runTraceAcceptor
+  ) where
+
+import qualified Codec.Serialise as CBOR
+import           Control.Concurrent.STM.TBQueue (TBQueue)
+import           Control.Exception.Extra (try_)
+import           Data.Typeable (Typeable)
+import           System.Time.Extra (sleep)
+
+import           Ouroboros.Network.Util.ShowProxy (ShowProxy(..))
+
+import           Trace.Forward.Network.Acceptor (listenToForwarder)
+import           Trace.Forward.Configuration (AcceptorConfiguration (..))
+import           Trace.Forward.Protocol.Type (NodeInfoStore)
+
+runTraceAcceptor
+  :: (CBOR.Serialise lo,
+      ShowProxy lo,
+      Typeable lo)
+  => AcceptorConfiguration lo -- ^ Acceptor configuration.
+  -> TBQueue lo               -- ^ The queue received 'TraceObject's will be written in.
+  -> NodeInfoStore            -- ^ The store node's basic info will be written in.
```

Comment:
I'm a little confused by the English of the comments.

Is the basic info being written to trace?